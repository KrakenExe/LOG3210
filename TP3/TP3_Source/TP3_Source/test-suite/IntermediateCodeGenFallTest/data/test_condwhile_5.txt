Declaration
num a;
bool b;
bool c;
num d;
EndDeclaration

d = 1;
while(b) {
  d = d + 1;
  while(c && true) {
    a = 42;
  }
}
  
