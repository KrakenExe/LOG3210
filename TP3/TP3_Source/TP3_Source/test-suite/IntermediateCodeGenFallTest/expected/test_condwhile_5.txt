d = 1
_L1
_L2
ifFalse b == 1 goto _L0
_t0 = d + 1
d = _t0
_L3
_L4
ifFalse c == 1 goto _L2
a = 42
goto _L4
goto _L2
_L0
